sw/device/tests/sim_dv/meson.build

```diff
@@ -53,9 +65,9 @@ spi_tx_rx_test_lib = declare_dependency(
     ],
   ),
 )
-
 sw_tests += {
-  'uart_tx_rx_test': uart_tx_rx_test_lib,
-  'gpio_test': gpio_test_lib,
-  'spi_tx_rx_test': spi_tx_rx_test_lib,
+  'spi_tx_rx_test': {
+    'library': spi_tx_rx_test_lib,
+    'dv_frames': false,
+  }
```

Comment:
Oh man, another rebase which went wrong. I'm rather certain I fixed this file as well when I changed the other one ... Thanks for fixing that up Sam!

Reply:
I think I haven't rebased onto `master` yet… oops.

Reply:
Ok, I have now, and this diff should look more reasonable.